docs/html/development/release-process.rst

```diff
@@ -18,6 +18,10 @@ that month will be up to the release manager for that release. If there are
 no changes, then that release month is skipped and the next release will be
 3 months later.
 
+The version number of pip is ``YY.N``, where ``YY`` is the year of the release
+and ``N`` identifies which release of the year (0=January, 1=April, 2=July, and
+3=October).
```

Comment:
@pfmoore If I trust [PyPI](https://pypi.org/project/pip#history), pip 24.0 was released in February and pip 24.1 in June, which doesn't quite match what you've written here, so perhaps it would be better to be less specific ? 😁

Reply:
The releases are the ones quoted in the existing paragraph above. They are described using the intended month of release but yes, we do slip sometimes (rather more often than would be ideal, but we're highly limited in resources). The problem with being less specific is that if we decide to *skip* a release (for example, the July release, as we've only just released 24.1) people would get equally confused that we had 24.0, 24.1 and 24.3 but no 24.2.

This is a compromise, to get the idea across without being over-complicated.

Reply:
It might be clearer to refer to them as quarters of the yeae?

Reply:
That would work, yes.

Reply:
How does https://github.com/pypa/pip/pull/12783 look?